url8e6440c8aa_robdimsdale_project_euler_tgz-pJ8-com_rmd_personal_projecteuler_MainJ8/src/com/rmd/personal/projecteuler/Main.java

```diff
@@ -0,0 +1,133 @@
+package com.rmd.personal.projecteuler;
+
+import com.rmd.personal.projecteuler.p1to19.P001;
+import com.rmd.personal.projecteuler.p1to19.P002;
+import com.rmd.personal.projecteuler.p1to19.P003;
+import com.rmd.personal.projecteuler.p1to19.P004;
+import com.rmd.personal.projecteuler.p1to19.P005;
+import com.rmd.personal.projecteuler.p1to19.P006;
+import com.rmd.personal.projecteuler.p1to19.P007;
+import com.rmd.personal.projecteuler.p1to19.P008;
+import com.rmd.personal.projecteuler.p1to19.P009;
+import com.rmd.personal.projecteuler.p1to19.P010;
+import com.rmd.personal.projecteuler.p1to19.P011;
+import com.rmd.personal.projecteuler.p1to19.P012;
+import com.rmd.personal.projecteuler.p1to19.P013;
+import com.rmd.personal.projecteuler.p1to19.P014;
+import com.rmd.personal.projecteuler.p1to19.P015;
+import com.rmd.personal.projecteuler.p1to19.P016;
+import com.rmd.personal.projecteuler.p1to19.P017;
+import com.rmd.personal.projecteuler.p1to19.P018;
+import com.rmd.personal.projecteuler.p1to19.P019;
+import com.rmd.personal.projecteuler.p20to39.P020;
+import com.rmd.personal.projecteuler.p20to39.P021;
+import com.rmd.personal.projecteuler.p20to39.P022;
+import com.rmd.personal.projecteuler.p20to39.P023;
+import com.rmd.personal.projecteuler.p20to39.P024;
+import com.rmd.personal.projecteuler.p20to39.P025;
+import com.rmd.personal.projecteuler.p20to39.P026;
+import com.rmd.personal.projecteuler.p20to39.P027;
+import com.rmd.personal.projecteuler.p20to39.P028;
+import com.rmd.personal.projecteuler.p20to39.P029;
+import com.rmd.personal.projecteuler.p20to39.P030;
+import com.rmd.personal.projecteuler.p20to39.P031;
+import com.rmd.personal.projecteuler.p20to39.P032;
+import com.rmd.personal.projecteuler.p20to39.P033;
+import com.rmd.personal.projecteuler.p20to39.P034;
+import com.rmd.personal.projecteuler.p20to39.P035;
+import com.rmd.personal.projecteuler.p20to39.P036;
+import com.rmd.personal.projecteuler.p20to39.P037;
+import com.rmd.personal.projecteuler.p20to39.P038;
+import com.rmd.personal.projecteuler.p20to39.P039;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public final class Main {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This class references 57 other classes. By comparison, 99% of the classes in the CodeGuru reference dataset reference fewer. This indicates that this class is highly coupled with other classes. A class that is highly coupled with other classes is difficult to understand and its behavior might change unexpectedly when one of its referenced classes is updated. High coupling could also increase the integration test complexity, maintenance cost and technical debt. We recommend that you simplify this class or break it into multiple classes.